ruby-kafka.gemspec

```diff
@@ -25,6 +25,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  spec.add_runtime_dependency 'gssapi', '>=1.2.0'
```

Comment:
yes. No need to install anything else if you don't plan to use gssapi sasl authentication.